app/src/main/java/org/oppia/android/app/utility/ClickableAreasImage.kt

```diff
@@ -109,6 +110,11 @@ class ClickableAreasImage(
           imageRect.height().roundToInt()
         )
         val newView = View(it.context)
+        // ClickableArea co-ordinates are not laid-out properly in RTL. The image region co-ordinates are
+        // from left-to-right and touch co-ordinates that draws frame are from right-to-left and it looks
+        // that the co-ordinates are always towards the right end in RTL which is creating problem.Thus to
+        // avoid this situation force layout direction to LTR
```

Comment:
```suggestion
        // ClickableArea coordinates are not laid-out properly in RTL. The image region coordinates are
        // from left-to-right with an upper left origin and touch coordinates from Android start from the right in RTL mode. Thus, to avoid this situation, force layout direction to LTR in all situations.
```

Cleaned up (needs to be re-wrapped at 100 chars per line though).

Reply:
Not quite done--please also fix the wrapping.

Reply:
Done.